pkg/pmem-state/pmme-state.go

```diff
@@ -0,0 +1,147 @@
+/*
+Copyright 2019  Intel Corporation.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+package pmemstate
```

Comment:
Typo in the filename.

Can you add unit tests to this package?

Reply:
Done